sites/partners/__tests__/pages/listings/index.test.tsx

```diff
@@ -71,6 +92,301 @@ describe("listings", () => {
+        return res(ctx.json({ items: [listing], meta: { totalItems: 1, totalPages: 1 } }))
+      }),
+      rest.get("http://localhost/api/adapter/listings", (_req, res, ctx) => {
+        return res(ctx.json({ items: [listing], meta: { totalItems: 1, totalPages: 1 } }))
+      }),
+      rest.get("http://localhost/api/adapter/user", (_req, res, ctx) => {
+        return res(
+          ctx.json({
+            id: "user1",
+            roles: { id: "user1", isAdmin: false, isPartner: true },
+          })
+        )
+      }),
+      rest.post("http://localhost:3100/auth/token", (_req, res, ctx) => {
+        return res(ctx.json(""))
+      })
+    )
+
+    render(
+      <AuthContext.Provider
+        value={{
+          initialStateLoaded: true,
+          profile: {
+            ...mockUser,
+            jurisdictions: [
+              {
+                id: "id1",
+                featureFlags: [
+                  {
+                    name: FeatureFlagEnum.enableUnitGroups,
+                    active: false,
+                  } as FeatureFlag,
+                ],
+              } as Jurisdiction,
+            ],
+          },
+        }}
+      >
+        <ListingsList />
+      </AuthContext.Provider>
+    )
+    expect(screen.getByText("Available units")).toBeDefined()
+    expect(screen.getByText("Open waitlist")).toBeDefined()
+  })
+
+  it("should not show is last updated column if feature flag is off", () => {
+    window.URL.createObjectURL = jest.fn()
+    document.cookie = "access-token-available=True"
+    server.use(
+      rest.get("http://localhost:3100/listings", (_req, res, ctx) => {
+        return res(ctx.json({ items: [listing], meta: { totalItems: 1, totalPages: 1 } }))
+      }),
+      rest.get("http://localhost/api/adapter/listings", (_req, res, ctx) => {
+        return res(ctx.json({ items: [listing], meta: { totalItems: 1, totalPages: 1 } }))
+      }),
+      rest.get("http://localhost/api/adapter/user", (_req, res, ctx) => {
+        return res(
+          ctx.json({
+            id: "user1",
+            roles: { id: "user1", isAdmin: false, isPartner: true },
+          })
+        )
+      }),
+      rest.post("http://localhost:3100/auth/token", (_req, res, ctx) => {
+        return res(ctx.json(""))
+      })
+    )
+
+    render(
+      <AuthContext.Provider
+        value={{
+          initialStateLoaded: true,
+          profile: {
+            ...mockUser,
+            jurisdictions: [
+              {
+                id: "id1",
+                featureFlags: [
+                  {
+                    name: FeatureFlagEnum.enableUnitGroups,
+                    active: true,
+                  } as FeatureFlag,
+                ],
+              } as Jurisdiction,
+            ],
+          },
+        }}
+      >
+        <ListingsList />
+      </AuthContext.Provider>
+    )
+    expect(screen.queryByText("Available units")).toBeNull()
```

Comment:
thought: looking at this made me think about how accessible this table is. I know you have mentioned it's not the greatest. for example the "name" of this columnheader  is  "Press Space to toggle all rows selection (unchecked) Available units". Ideally once we get this table more accessible we would query by role instead text

```
      <div
        aria-colindex="6"
        class="ag-header-cell ag-focus-managed"
        col-id="unitsAvailable"
        role="columnheader"
        style="width: 120px; left: 850px;"
        tabindex="-1"
        unselectable="on"
      />
```